src/devices/Arduino/Arduino.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <RootNamespace>Iot.Device.Arduino</RootNamespace>
+  </PropertyGroup>
+
+ <ItemGroup>
+    <Compile Include="*.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj" />
+    <ProjectReference Include="..\Common\CommonHelpers.csproj" />
+    <PackageReference Include="System.IO.Ports" Version="4.7.0" />
```

Comment:
Can we move these two package references to eng\Versions.props as properties so that we reference the same version across the repo?

Reply:
Sure, they actually already are there.